coalib/parsing/ConfParser.py

```diff
@@ -32,6 +32,23 @@ def __init__(self,
         self.__rand_helper = None
         self.__init_sections()
 
+    def check_valid_lines(self, lines):
+        """
+        This function checks for some invalid config file commands like
+        ``default_action: *: ShowPatchAction``
+        which show error in the previous line. This function gives the
+        correct error in such cases.
+        """
+        for line in lines:
+            occ = line.find(':')
+            line_substr = line[:occ]
+            occ_equals = line_substr.find('=')
+            occ_comment = line_substr.find('#')
+            if occ != -1 and not ((occ_equals != -1 and occ_equals < occ) or
```

Comment:
I expect this approach is going to fail on

```
default_action =
  *: ShowPatchAction
```
(which I assume is currently valid)

and why cant I have?

```
a = b
# 1:1 relationship
```
(again, I assume the the parser currently understands that correctly)